com.unity.netcode.gameobjects/Runtime/Timing/NetworkTimeSystem.cs

```diff
@@ -257,7 +263,7 @@ public void Reset(double serverTimeSec, double rttSec)
             Sync(serverTimeSec, rttSec);
             Advance(0);
         }
-
+        internal int SyncCount;
```

Comment:
Out of interest, what is this count for?

Reply:
This is an internal counter that can be useful for debugging purposes and might be useful at a later time to provide users with additional debugging tools. Basically, this gets incremented every time a client receives a `TimeSyncMessage`... which can be checked for changes...which then provides one with the ability to determine the actual local time frequency of receiving and processing time sync messages. 
_(I use it in the added debug stuff within `NetworkTransform` to know on what time synch message count a state update was received as well to be able to check if there is any variance in the time to target for a state update that occurs within the same approximate time frame that a time synchronization occurs)_